tests/test_mapper.py

```diff
@@ -0,0 +1,66 @@
+import sys
+from typing import Optional, Type
+
+import pytest
+
+from gdcmodels import mapper, utils
+
+if sys.version_info < (3, 9):
+    import importlib_resources as resources
+else:
+    from importlib import resources
+
+
+@pytest.fixture
+def mock_mapper():
+    mapping_file = resources.files("tests") / "data/mapper/mapping.yaml"
+    mapping = utils.load_yaml(mapping_file.read_bytes())
```

Comment:
Since we are keeping the `utils` feature, you should consider making it more specific.`utils` easily become a dumping ground for things that do not fit in anywhere else and based on the current usage you want a customized yaml loader. This can be put into a more specific module or class rather than a generic `utils` module.